runner/generate_helper_sql.go

```diff
@@ -45,12 +45,7 @@ func GenerateStatsHelperSql(ctx context.Context, server *state.Server, globalCol
 	}
 	defer db.Close()
 
-	version, err := postgres.GetPostgresVersion(ctx, logger, db)
-	if err != nil {
-		return "", fmt.Errorf("error collecting Postgres version: %s", err)
-	}
```

Comment:
Are these removed because we're moving the version check earlier in later patches?

Reply:
This was removed because we don't actually use the version information anywhere in this method - it was presumably done before because `GetDatabases` unnecessarily had a `version` argument (that it did not use).